README.md

```diff
@@ -50,7 +50,7 @@ drush potion-export [--non-customized] [--customized] [--untranslated] [--progre
 ### Generate, scrape & parse your code to generate po file
 
 ```bash
-drush potion-generate [--exclude-yaml] [--exclude-twig] [--exclude-php] [--progress] [--verbose] [--quiet] langcode source dest
+drush potion-generate [--exclude-yaml] [--exclude-twig] [--exclude-php] [--recursive] [-q|--quiet] [-h|--help] langcode source dest
```

Comment:
I think it's could be great to have the description of the arguments in the `README.md` like `ROADMAP.md`. 
E.g: 
`
*Parameters*
- `source`: The source po file
- `langcode`: Import the po into the given langcode
- `-h|--help`: Display usage details.
...
`